src/app/shared/form/builder/ds-dynamic-form-ui/models/scrollable-dropdown/dynamic-scrollable-dropdown.component.ts

```diff
@@ -88,6 +92,11 @@ export class DsDynamicScrollableDropdownComponent extends DsDynamicVocabularyCom
       this.selectedIndex = 0;
       this.cdr.detectChanges();
     });
+
+    this.group.get(this.model.id).valueChanges.pipe(distinctUntilChanged())
+      .subscribe((value) => {
+        this.setCurrentValue(value);
+      });
```

Comment:
I would move this part to the end of `ngOnInit()`, otherwise this subscription will be created many times.